PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1644,6 +1644,17 @@ bool FindReplaceDlg::processReplace(const TCHAR *txt2find, const TCHAR *txt2repl
 				msg += moreMatches?TEXT("The next occurence found"):TEXT("The next occurence not found");
 				setStatusbarMessage(msg, FSMessage);
 			}
+
+			if (pTextFind)
+			{
+				delete[] pTextFind;
+				pTextFind = nullptr;
+			}
+			if (pTextReplace)
+			{
+				delete[] pTextReplace;
+				pTextReplace = nullptr;
+			}

```

Comment:
You're right about pTextFind. But pTextReplace is required because of statment
```C++
Searching::convertExtendedToString(pTextReplace, pTextReplace, stringSizeReplace);
```
However, I have updated PR for pTextFind.